modules/loop_search/loop_search.features.inc

```diff
@@ -40,7 +40,6 @@ function loop_search_default_search_api_index() {
             "default" : "0",
             "bundles" : {
               "external_sources" : "external_sources",
-              "leaf" : "leaf",
```

Comment:
Shouldn't this be part of the previous PR (https://github.com/os2loop/profile/pull/113)?